aws-sdk-resources/lib/aws-sdk-resources/services/s3/encryption/io_decrypter.rb

```diff
@@ -31,6 +32,53 @@ def reset_cipher
         end
 
       end
+
+      # @api private
+      class IOAuthDecrypter
```

Comment:
Yup. I wrote the code original in the one file for convience and failed to remove it when I added it to its own file. I'll clean that up.